browser/src/control/jsdialog/Definitions.Menu.ts

```diff
@@ -540,4 +598,42 @@ menuDefinitions.set('ChangeSlideLayoutMenu', [
 	{ type: 'separator' }, // required to show dropdown arrow
 ] as Array<MenuDefinition>);
 
+menuDefinitions.set(
+	'PictureBrightness',
+	generatePictureBrightnessMenu('GrafLuminance'),
+);
+
+menuDefinitions.set(
+	'PictureContrast',
+	generatePictureContrastMenu('GrafContrast'),
+);
+
+menuDefinitions.set(
+	'PictureTransparency',
+	generatePictureTransparencyMenu('GrafTransparence'),
+);
+
+menuDefinitions.set('PictureColorMode', [
+	{
+		text: 'Default',
+		img: 'insertgraphic',
+		uno: '.uno:GrafMode?ColorMode:short=0',
+	},
+	{
+		text: 'Grayscale',
+		img: 'insertgraphic',
+		uno: '.uno:GrafMode?ColorMode:short=1',
+	},
+	{
+		text: 'Black/White',
+		img: 'insertgraphic',
+		uno: '.uno:GrafMode?ColorMode:short=2',
+	},
+	{
+		text: 'Watermark',
```

Comment:
translation: `_('Watermark')`

here and in other places

Reply:
Done